docs/odh/oci-model-storage.md

```diff
@@ -8,15 +8,228 @@ which also explains how to deploy models from OCI images.
+stored in the OCI image referred by the `storageUri` field. Check the status
+of the deployment with the following command:
+```shell
+oc get inferenceservice
+
+NAME                    URL                                                       READY   PREV   LATEST   PREVROLLEDOUTREVISION   LATESTREADYREVISION                     AGE
+sample-isvc-using-oci   https://sample-isvc-using-oci-oci-model-example.example   True           100                              sample-isvc-using-oci-predictor-00001   2m11s
+```
+
+> [!IMPORTANT]
+> Remember that, by default, models are exposed outside the cluster and not
+> protected with authorization. Read the [authorization guide](authorization.md#deploying-a-protected-inferenceservice)
+> and the [private services guide (TODO)](#TODO) to learn how to privately deploy
+> models and how to protect them with authorization.
+
+Test the model is working:
+```sh
+# Query
+curl https://sample-isvc-using-oci-oci-model-example.apps.rosa.ehernand-test.v16g.p3.openshiftapps.com/v1/completions \
+      -H "Content-Type: application/json" \
+      -d '{
+          "model": "sample-isvc-using-oci",
+          "prompt": "What is the IBM granite-3 model?",
+          "max_tokens": 200,
+          "temperature": 0.8
+      }' | jq
+
+# Response:
+{
+  "id": "cmpl-639e7e1911e942eeb34bc9db9ff9c9fc",
+  "object": "text_completion",
+  "created": 1733527176,
+  "model": "sample-isvc-using-oci",
+  "choices": [
+    {
+      "index": 0,
+      "text": "\n\nThe IBM Granite-3 is a high-performance computing system designed for complex data analytics, artificial intelligence, and machine learning applications. It is built on IBM's 
+        Power10 processor-based architecture and features:\n\n1. **Power10 Processor-based Architecture**: The IBM Granite-3 is powered by IBM's latest Power10 processor, which provides high performance
+        , energy efficiency, and advanced security features.\n\n2. **High-Performance Memory**: The system features high-speed memory, enabling fast data access and processing, which is crucial for comp
+        lex data analytics and AI applications.\n\n3. **Large Memory Capacity**: The IBM Granite-3 supports large memory capacities, allowing for the processing of vast amounts of data.\n\n4. **High-Spe
+        ed Interconnect**: The system features a high-speed interconnect, enabling quick data transfer between system components.\n\n5. **Advanced Security Features",
+      "logprobs": null,
+      "finish_reason": "length",
+      "stop_reason": null,
+      "prompt_logprobs": null
+    }
+  ],
+  "usage": {
+    "prompt_tokens": 10,
+    "total_tokens": 210,
+    "completion_tokens": 200
+  }
+}
+```
+
+## Creating and deploying an OCI image of MobileNet v2-7 model
+
+The MobileNet v2-7 model is available at the [onnx/models](https://github.com/onnx/models/tree/main/validated/vision/classification/mobilenet)
+GitHub repository. This model is in ONNX format.
```

Comment:
maybe let the reader know that the model needs to be downloaded before hand.

Reply:
Downloading the model is part of the process.

Reply:
yep, but differently from the first example, it is a manual process, maybe a small note about it might improve the user experience.

Reply:
I've added ` and in this example you will download it.` to the text.